mmv1/third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -34,28 +34,28 @@ func TestBigQueryTableSchemaDiffSuppress(t *testing.T) {
 			ExpectDiffSuppress: false,
 		},
 		"no change": {
-			Old:                "[{\"name\": \"someValue\", \"anotherKey\" : \"anotherValue\", \"finalKey\" : {} }]",
-			New:                "[{\"name\": \"someValue\", \"anotherKey\" : \"anotherValue\", \"finalKey\" : {} }]",
+			Old:                "[{\"name\": \"someValue\", \"type\": \"INT64\", \"anotherKey\" : \"anotherValue\", \"finalKey\" : {} }]",
```

Comment:
This change is to reflect the actual restrictions in the API and not new validation introduced in this PR, right? That's what I understood "Also removed tests for setting type to nil, because a field type being nil is invalid." to mean, at least

Reply:
yeah, the type field is actually required in the API, but a lot of these unit test cases were either missing it or explicitly testing what happened when you deleted it. (But in reality, the API doesn't allow you to leave out the type field.) If we make sure it's present, we don't have to check for nil values.

Reply:
hmmm... I might need to double-check whether setting an invalid schema would cause a panic, actually.